zebra-chain/src/parameters/network/testnet.rs

```diff
@@ -232,6 +232,9 @@ pub struct ParametersBuilder {
     target_difficulty_limit: ExpandedDifficulty,
     /// A flag for disabling proof-of-work checks when Zebra is validating blocks
     disable_pow: bool,
+    /// Whether to allow transactions with transparent outputs that spend coinbase inputs,
```

Comment:
Only outputs are "spendable", not inputs.

```suggestion
    /// Whether to allow transactions with transparent outputs to spend coinbase outputs,
```